docs/topics/class-based-views/generic-display.txt

```diff
@@ -402,7 +411,8 @@ First, we'd need to add an author detail bit in the URLconf to point to a
 custom view::
 
     from django.urls import path
-    from books.views import AuthorDetailView
+
+    from .views import AuthorDetailView
```

Comment:
Leave alone. This looks like a root URLConf since it uses "authors/...".

Reply:
ok